Misc/NEWS.d/next/Library/2019-02-07-16-22-50.bpo-35931._63i7B.rst

```diff
@@ -0,0 +1 @@
+pdb: handle SyntaxError with ``debug``.
```

Comment:
I will change for: "Add handle  SyntaxError with ``debug`` on pdb" (just an opinon)